orangecontrib/bio/ontology.py

```diff
@@ -64,6 +64,7 @@
 except NameError:
     basestring = str
     intern = sys.intern
+    from io import open
```

Comment:
The following two options are more appropriate in order to avoid imports not located at the start of the file and overriding names provided by default namespace: 
1. `import io` + call `io.open(...)`
2. `from io import open as myopen` + `call myopen(...)`